packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -2276,11 +2276,11 @@ class TestWidgetInspectorService extends Object with WidgetInspectorService {
       );
     }, skip: isBrowser);
 
-    testWidgets('ext.flutter.inspector.structuredErrors', (WidgetTester tester) async {
```

Comment:
I have to change this part because the testWidget overrides the onError directly which is essential to make sure present error is called. I think this change is fine because the structure error should only be applicable in non widget test environment.

Reply:
I previously attempt to make testWidget overrides presentError as well, but it was a bad idea because it will bleed a lot of error catching logic into the next test. Furthermore, it does not make sense for testWidget to override the presentError because it is not changing the way the error is **presenting** to user. It is changing the way how framework **handling** the error.